template/Content/.template.config/template.json

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "http://json.schemastore.org/template",
+    "author": "David Fowler",
+    "classifications": [ "Web", "ASP.NET", "FeatherHttp" ],
+    "identity": "FeatherHttp.Templates",
+    "name": "FeatherHttp Templates",
+    "description": "Am ultra lightweight ASP.NET project starter",
```

Comment:
typo: Am -> An

Reply:
Remove ASP.NET, put .NET Core

Reply:
Lightweight low ceremony API for ASP.NET Core services.

Reply:
Okay, I've fixed that typo